orttraining/orttraining/python/deprecated/training_session.py

```diff
@@ -38,8 +38,20 @@ def __del__(self):
     def get_state(self):
         return self._sess.get_state()
 
+    def get_model_state(self, include_mixed_precision_weights=False):
+        return self._sess.get_model_state(include_mixed_precision_weights)
+    
+    def get_optimizer_state(self):
+        return self._sess.get_optimizer_state()
+
+    def get_partition_info_map(self):
+        return self._sess.get_partition_info_map()
+
     def load_state(self, dict, strict=False):
         self._sess.load_state(dict, strict)
+    
+    def load_model_optimizer_state(self, model_state, opt_state, strict=False):
```

Comment:
@baijumeswani is this functionality needed? As far as I understand, the model states will be loaded into the onnx graph and training session will be re-initialized(by passing the optim state to be loaded) for loading the checkoint.

Reply:
So, this functionality is not directly needed. As we discussed, at the moment we are looking to create a new training session for each load state dict call.
If the trainer configuration is the same before and after migration, creating a new training session may not be required. In that case, this function will be useful. But as of now, we will not be using this function.